Sources/swift-build/--init.swift

```diff
@@ -36,7 +36,8 @@ final class InitPackage {
     private func writeManifestFile() throws {
         let manifest = Path.join(rootd, Manifest.filename)
         guard manifest.exists == false else {
-            throw Error.ManifestAlreadyExists
```

Comment:
I feel it's better to throw an error here instead of calling return, if Manifest already exists.